src/test/mir-opt/simplify-arm.rs

```diff
@@ -1,9 +1,12 @@
-// compile-flags: -Z mir-opt-level=1
+// compile-flags: -Z mir-opt-level=2
```

Comment:
I changed this to 2 since that is the level that SimplifyArmIdentity runs at. The new pass SimplifyDiscriminantArm currently has no mir-opt-level gate. Should it? AFAICT it SimplifyDiscriminantArm does not make much sense by itself. It needs SimplifyArmIdentity (gated at level 2+) and SimplifyBranchSame (not gated).